docs/tutorial/typesfuns.rst

```diff
@@ -538,9 +538,10 @@ Note: Declaration Order and ``mutual`` blocks
 
 In general, functions and data types must be defined before use, since
 dependent types allow functions to appear as part of types, and their
-reduction behaviour to affect type checking. However, this restriction
-can be relaxed by using a ``mutual`` block, which allows data types
-and functions to be defined simultaneously:
+reduction behaviour to affect type checking (though this is only true of
```

Comment:
I guess I'm not sure how to word it better/more clearly. Any suggestions? Or @ahmadsalim, do you have any thoughts on this?